geolocator_platform_interface/test/src/implementations/method_channel_geolocator_test.dart

```diff
@@ -134,6 +134,46 @@ void main() {
       });
     });
 
+    group('getLocationAccuracy: When requesting the Location Accuracy Status',
+        () {
+      test(
+          'Should receive reduced accuracy if Location Accuracy is reduced',
+          () async {
+        // Arrange
+        MethodChannelMock(
+          channelName: 'flutter.baseflow.com/geolocator',
+          method: 'getLocationAccuracy',
+          result: 0,
+        );
+
+        // Act
+        final locationAccuracy =
+            await MethodChannelGeolocator().getLocationAccuracy();
+
+        // Assert
+            expect(locationAccuracy, LocationAccuracyStatus.reduced);
```

Comment:
nit: small formatting error here:

```suggestion
        // Assert
        expect(locationAccuracy, LocationAccuracyStatus.reduced);
```